lib/cs_guide/resources/venue.ex

```diff
@@ -53,6 +53,8 @@ defmodule CsGuide.Resources.Venue do
     )
 
     has_many(:venue_images, CsGuide.Images.VenueImage)
+    # Not sure that they should have many? >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+    has_many(:discount_codes, CsGuide.DiscountCode)
```

Comment:
@Cleop it's _almost_ guaranteed that many discount codes will be a thing.
_Most_ brands will want to use _multiple_ "_channels_" and to _track_ them.
E.g: a Promo on Facebook (Online) will be different to one in a Festival (RL)
Discount code tracking data will be the "gold" of CS in the future. 💰